Client/game_sa/CModelInfoSA.cpp

```diff
@@ -1367,6 +1368,46 @@ void CModelInfoSA::ResetSupportedUpgrades()
     m_ModelSupportedUpgrades.Reset();
 }
 
+void CModelInfoSA::SetObjectPropertiesGroup(unsigned short usNewGroup)
+{
+    unsigned short usOrgGroup = GetObjectPropertiesGroup();
+    if (usOrgGroup == usNewGroup)
+        return;
+
+    if (!MapFind(ms_OriginalObjectPropertiesGroups, m_dwModelID))
+        MapSet(ms_OriginalObjectPropertiesGroups, m_dwModelID, usOrgGroup);
+
+    GetInterface()->usDynamicIndex = usNewGroup;
+}
+
+unsigned short CModelInfoSA::GetObjectPropertiesGroup()
+{
+    unsigned short usGroup = GetInterface()->usDynamicIndex;
+    if (usGroup == 0xFFFF)

```

Comment:
What is the meaning of a `0xFFFF (-1)` property group?

Reply:
If i remember correctly, it's used by the game to determine that this object id doesn't have any custom physical properties assigned to it, in such cases the default(0) physical properties group is used for most physical calculations on those objects